migrations/Version202110280904504106_pciSamples.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\pciSamples\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\Exception\IrreversibleMigration;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\pciSamples\scripts\install\RegisterPciTextReaderIMS;
+use oat\qtiItemPci\model\PciModel;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version202110280904504106_pciSamples extends AbstractMigration
+{
+
+    public function getDescription(): string
+    {
+        return 'Add basic keayboard navigation to the TextReaderInteraction';
```

Comment:
Small typo: `keayboard `
```suggestion
        return 'Add basic keyboard navigation to the TextReaderInteraction';
```